include/tss2/tss2_tcti.h

```diff
@@ -5,6 +5,31 @@
  * Copyright 2015, Andreas Fuchs @ Fraunhofer SIT
  *
  * All rights reserved.
+ *
+ * Copyright (c) 2019, Wind River Systems.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
You don't need to add the BSD-Clause in again, there is a SPDX-Identifier at the top of the file.
Or do you think that that is not sufficient to reflect your choice of license ?

Reply:
Yes, the license needs to be shown here explicit. This is required by Wind River.